remoto/log.py

```diff
@@ -12,7 +12,9 @@ def reporting(conn, result, timeout=None):
         try:
             received = result.receive(timeout)
             level_received, message = list(received.items())[0]
-            log_map[level_received](message.strip('\n'))
+            message = message.splitlines()
```

Comment:
To correct the copy/paste error, the 3rd change is as below:
            message = message.splitlines()
            for i in range(len(message)):
                log_map[level_received](message[i].strip("'"))